gym/lib/gym/runner.rb

```diff
@@ -12,28 +12,31 @@ def run
         build_app
       end
       verify_archive
-      FileUtils.mkdir_p(File.expand_path(Gym.config[:output_directory]))
+      output_path = File.expand_path(Gym.config[:output_directory])
+      FileUtils.mkdir_p(output_path)
 
       if Gym.project.ios? || Gym.project.tvos?
         fix_generic_archive # See https://github.com/fastlane/fastlane/pull/4325
+        if Gym.config[:skip_package_ipa]
+          return BuildCommandGenerator.archive_path
```

Comment:
This is good like it is, but you for short one line statements, you can use Ruby:
```ruby
return BuildCommandGenerator.archive_path if Gym.config[:skip_package_ipa]
```
which makes it very natural to read the code